code/src/components/Task.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+
+import { todos } from '../reducers/todos'
+// need to import the reducer we dispatch our action to
+
+const ListItem = styled.li`
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 0;
+  height: 24px;
+  width: 300px;
+  list-style: none;
+  border-bottom: 1px solid black;
+  text-align: left;
+  font-size: 20px;
+`
+
+const Wrapper = styled.div`
+  margin-top: -3px;
```

Comment:
Almost always there is a better solution than negative margin. In your case, it would be :

```
display: flex;
align-items: center;
```